.github/workflows/build.yml

```diff
@@ -30,20 +30,29 @@ jobs:
           git tag --list
 
       - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
-          java-version: 1.8
+          java-version: '8'
+          distribution: 'zulu'
 
       - name: Cache Maven packages
-        uses: actions/cache@v2.1.4
+        uses: actions/cache@v3.3.2
         with:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
 
+      - uses: docker/setup-qemu-action@v2
+        with:
+          platforms: "arm64"
+
       - name: Build and test
         run: mvn test -B
 
+      # The skipped tests is because h2o-xgboost isn't ready to use on arm64
```

Comment:
Is there any assertion somewhere that prevent users to use `h2o-xgboost` with `arm64`.

Reply:
no, there isn't